src/rpc/blockchain.cpp

```diff
@@ -1778,6 +1778,11 @@ void CalculatePercentilesByWeight(CAmount result[NUM_GETBLOCKSTATS_PERCENTILES],
     }
 }
 
+void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fIncludeHex)
```

Comment:
There is only one invocation of this function in this .cpp file. Maybe just inline it?